spec/requests/user_spec.rb

```diff
@@ -17,5 +17,10 @@
       @user.post_counter = -1
       expect(@user).to_not be_valid
     end
+
+    it 'PostCounter must be greater than or equal to zero' do
+      @user.post_counter = 7
+      expect(@user).to be_valid
+    end
   end
 end
```

Comment:
- [ ] Good job adding some tests but please kindly include integration tests for `User` index and show page with the following requirements.

### Index Page
- Tests that assert you can see the username of all other users.
- Tests that assert you can see the profile picture for each user.
- Tests that assert you can see the number of posts each user has written.
- Tests that redirect to the user's show page when you click on a user.

### Show Page
* Tests that assert you can see the user's profile picture.
* Tests that assert you can see the user's username.
* Tests that assert you can see the number of posts the user has written.
* Tests that assert you can see the user's bio.
* Tests that assert you can see the user's first 3 posts.
* Tests that assert you can see a button that lets me view all of a user's posts.
* Tests that redirect to the post's show page when you click on a user's post.
* Tests that redirect to the user's post's index page when you click on a **see all posts**.